src/printf.py

```diff
@@ -5,6 +5,6 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print("x is %i, y is %.5f, z is '%s'" % (x, y , z))
 
 # Use the 'format' string method to print the same thing
```

Comment:
Don't forget to check out this other suggested method of achieving the same result. It's pretty useful!